frontend/src/components/inbox/shared/InboxList.vue

```diff
@@ -0,0 +1,276 @@
+import IconSubject from "~icons/ic/baseline-subject"
+import IconNote from "~icons/ic/outline-comment-bank"
+import IconEdit from "~icons/ic/outline-edit"
+import IconView from "~icons/ic/outline-remove-red-eye"
+import IconClock from "~icons/ic/outline-watch-later"
+
+const props = defineProps<{
+  pageEntries?: Page<DocumentUnitListEntry>
+}>()
+
+const emit = defineEmits<{
+  updatePage: [number]
+  deleteDocumentationUnit: [documentUnitListEntry: DocumentUnitListEntry]
+  takeOverDocumentationUnit: [documentUnitListEntry: DocumentUnitListEntry]
+}>()
+
+const entries = computed(() => {
+  return props.pageEntries?.content || []
+})
+
+const showDeleteModal = ref(false)
+const selectedDocumentUnitListEntry = ref<DocumentUnitListEntry>()
+const popupModalText = computed(
+  () =>
+    `Möchten Sie die Dokumentationseinheit ${selectedDocumentUnitListEntry?.value?.documentNumber} wirklich dauerhaft löschen?`,
+)
+const scrollLock = useScrollLock(document)
+
+/**
+ * Clicking on a delete icon of a list entry shows a modal, which asks for user input to proceed
+ * @param {DocumentUnitListEntry} documentUnitListEntry - The documentationunit list entry to be deleted
+ */
+function showDeleteConfirmationDialog(
+  documentUnitListEntry: DocumentUnitListEntry,
+) {
+  selectedDocumentUnitListEntry.value = documentUnitListEntry
+  showDeleteModal.value = true
+}
+
+/**
+ * Propagates delete event to parent and closes modal again
+ */
+function onDelete() {
+  if (selectedDocumentUnitListEntry.value) {
+    emit("deleteDocumentationUnit", selectedDocumentUnitListEntry.value)
+    showDeleteModal.value = false
+  }
+}
+
+watch(showDeleteModal, () => (scrollLock.value = showDeleteModal.value))
+</script>
+
+<template>
+  <div data-testId="inbox-list">
+    <PopupModal
+      v-if="showDeleteModal"
+      aria-label="Dokumentationseinheit löschen"
+      :content-text="popupModalText"
+      header-text="Dokumentationseinheit löschen"
+      primary-button-text="Löschen"
+      primary-button-type="destructive"
+      @close-modal="showDeleteModal = false"
+      @primary-action="onDelete"
+    />
+    <Pagination
+      navigation-position="bottom"
+      :page="pageEntries"
+      @update-page="emit('updatePage', $event)"
+    >
+      <DataTable
+        class="ris-label2-bold text-gray-900"
+        :pt="{
+          thead: {
+            style: 'box-shadow: inset 0 -2px #DCE8EF;',
+          },
+        }"
+        :value="entries"
+      >
+        <Column selection-mode="multiple"></Column>
+        <Column field="documentNumber" header="Dokumentnummer">
+          <template #body="{ data: item }">
+            <div class="flex flex-row items-center space-x-8">
+              <div>{{ item.documentNumber }}</div>
+
+              <Tooltip text="Tooltip Text">
+                <IconAttachedFile
+                  class="flex-end h-20 w-20"
+                  :class="
+                    item.hasAttachments ? 'text-blue-800' : 'text-gray-500'
```

Comment:
```suggestion
                    item.hasAttachments ? 'text-blue-800' : 'text-blue-500'
```
Im Design ist es blue-500